_pytest/junitxml.py

```diff
@@ -71,6 +71,8 @@ def __init__(self, nodeid, xml):
         self.nodes = []
         self.testcase = None
         self.attrs = {}
+        self.testsuite_properties = {}
```

Comment:
Hmm I took a better look, I think this should actually go into the `LogXML` class instead, as it represents the entire XML file that will be written.